### ISSUE TITLE
Gauss LAZYFIND doesn't seems to do anything anymore

### ISSUE THREAD

**Issue by Bobbanz1** [BeeStation/NSV13#2263]
## Reproduction:
Issue discovered whilst doing something for Sophie's Broadside PR, I also managed to reproduce the issue in the master branch on my own fork so it's something that's in our current master branch

I have found this issue occurring here
https://github.com/BeeStation/NSV13/blob/a68b9633c95f7477bbdbdf897f3082506d85dae6/nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/gauss_gun.dm#L414

Instead of stopping the player from trying to load an empty crates worth of content into the gauss rack, it merely just continues on like there wasn't any conditional statement supposedly preventing people from loading an empty crate!

How to reproduce this issue:
Grab an empty crate.
Click drag it onto the Gauss Rack.
Watch as a progress bar appears and no mentions of "There's nothing in this crate you can load onto the gauss rack" is in sight.

**Comment by Bobbanz1** [comment]
https://user-images.githubusercontent.com/59128051/210432562-1b91cbd0-a882-418b-a019-9f90c242f5f3.mp4

**Comment by Bokkiewokkie** [comment]
Yeah this is because lazyfind only works for lazylists which contents obviously are not
